zhaquirks/tuya/air/ts0601_smart_air.py

```diff
@@ -0,0 +1,137 @@
+"""Tuya Smart Air House keeper."""
+
+from zigpy.profiles import zgp, zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya.air import (
+    TuyaAirQualityCO2,
+    TuyaAirQualityFormaldehyde,
+    TuyaAirQualityHumidity,
+    TuyaAirQualityPM25,
+    TuyaAirQualityTemperature,
+    TuyaAirQualityVOC,
+    TuyaCO2ManufCluster,
+)
+
+
+class TuyaSmartAirSensor(CustomDevice):
+    """Tuya Smart air house keeper device."""
+
+    signature = {
+        # NodeDescriptor(logical_type=<LogicalType.Router: 1>, complex_descriptor_available=0, user_descriptor_available=0, reserved=0, aps_flags=0, frequency_band=<FrequencyBand.Freq2400MHz: 8>, mac_capability_flags=<MACCapabilityFlags.FullFunctionDevice|MainsPowered|RxOnWhenIdle|AllocateAddress: 142>, manufacturer_code=4417, maximum_buffer_size=66, maximum_incoming_transfer_size=66, server_mask=10752, maximum_outgoing_transfer_size=66, descriptor_capability_field=<DescriptorCapability.NONE: 0>, *allocate_address=True, *is_alternate_pan_coordinator=False, *is_coordinator=False, *is_end_device=False, *is_full_function_device=True, *is_mains_powered=True, *is_receiver_on_when_idle=True, *is_router=True, *is_security_capable=False)
+        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=81, device_version=1,
+        # input_clusters=[0, 4, 5, 1026, 1029, 1037, 1067, 1070, 61184],
+        # output_clusters=[25, 10])
+        MODELS_INFO: [
+            ("_TZE200_mja3fuja", "TS0601"),
+            ("_TZE200_dwcarsat", "TS0601"),
+        ],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.SMART_PLUG,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    TuyaCO2ManufCluster.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
+            }
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT,
```

Comment:
Noticed `ON_OFF_LIGHT` as a type when reviewing. The previous/other Tuya "air-related" quirks also use this, so I guess it's fine/there's a reason, but we might wanna take a look at this in the future again.